fonts.css

```diff
@@ -0,0 +1,128 @@
+@font-face {
+    font-family: "Qanelas Soft";
+    src: url('./static/fonts/qanelas-soft/QanelasSoftBlack.otf') format('opentype');
```

Comment:
Is it working with ./static in path?
Because the fonts folder is created outside the static folder 🤔

Reply:
Yes :grimacing: